public/index.html

```diff
@@ -3,27 +3,56 @@
 	<head>
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<link rel="icon" href="favicon.ico">
+		<link rel="icon" href="assets/image/favicon.ico">
 
 		<title>OpenFisca - Turn law into software</title>
 
-		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
 	</head>
 	<body>
-		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
-
-		<h1>Turn law into software</h1>
-
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
-
-		<h3>Calculate citizens rights</h3>
-		<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
-
-		<h3>Predict company taxes</h3>
-		<p>Model any kind of entity and evaluate their eligibility to fiscal obligations and benefits.</p>
-
-		<h3>Assess the impact reforms could have</h3>
-		<p>Compare law as it is now and as it could be.</p>
+		<header>
+			<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
+
+			<h1>Turn law into software</h1>
+
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		</header>
+
+		<section>
+			<div>
+				<h3>Calculate citizens' rights</h3>
+				<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
+			</div>
+
+			<figure>
+				<video class="screenshot" src="assets/video/mes-aides.mp4" poster="assets/image/mes-aides.png" controls></video>
+				<figcaption><a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> example uses the OpenFisca Web API to calculate OpenFisca-France benefits.</figcaption>
```

Comment:
Mes Aides uses the OpenFisca Web API to inform French citizens on their eligibility to national and local benefits.